src/components/menu/__internal__/submenu/submenu.component.js

```diff
@@ -214,7 +227,11 @@ const Submenu = React.forwardRef(
               <SubmenuContext.Provider
                 value={{
                   isFocused: submenuFocusIndex === index,
+                  focusIndex: submenuFocusIndex,
                   handleKeyDown,
+                  blockIndex: React.Children.toArray(children).findIndex(
```

Comment:
Would this not only support a single `ScrollableBlock` within a `Submenu`?

Reply:
It would break it if there was multiple, my assumption here is what you said there, that there will only be a single scrollable block in the submenu - which I think is a fair assumption to make!

Reply:
Added a description to the MDX as discussed elsewhere